lib/operations/collection_ops.js

```diff
@@ -444,6 +571,171 @@ function ensureIndex(coll, fieldOrSpec, options, callback) {
   ensureIndexDb(coll.s.db, coll.s.name, fieldOrSpec, options, callback);
 }
 
+/**
+ * Creates a cursor for a query that can be used to iterate over results from MongoDB
+ *
+ * @method
+ * @param {object} [query={}] The cursor query object.
+ * @param {object} [options] Optional settings. See Collection.prototype.find for a list of options.
+ * @throws {MongoError}
+ * @return {Cursor}
+ */
+const findSchema = {
```

Comment:
again I think this doesn't quite belong here, and duplicates the existing schema. At very least we should move the schemas somewhere we can reuse them, but also I'm not quite sure I understand why we need to reapply this here.